### PR TITLE
Update simulator

### PR DESCRIPTION
**Pull request by andrewheusser**
-we removed the simulatorFactory because its now built into the board
by default

-moved the stream buttons to the dashboard page

### CODE REVIEW

**Comment by andrewjaykeller** [comment]
Yeaaaaa!

File: client/components/board/boardFactory.js [hunk 1]
```diff
@@ -3,7 +3,11 @@ angular.module('OpenEXP')
 
         // requirements
         const OpenBCIBoard = require('openbci-sdk');
-        const ourBoard = new OpenBCIBoard.OpenBCIBoard();
+        const ourBoard = new OpenBCIBoard.OpenBCIBoard({
+          verbose:true
```
Comment (andrewjaykeller) [inline client/components/board/boardFactory.js]: do you want to include this verbose?

File: client/components/board/boardFactory.js [hunk 2]
```diff
@@ -39,16 +43,25 @@ angular.module('OpenEXP')
 
         // connect to the board and start streaming if no errors
         var connect = (portName) => {
+          return new Promise((res, rej) => {
             ourBoard.connect(portName)
                 .then(boardSerial => {
```
Comment (andrewjaykeller) [inline client/components/board/boardFactory.js]: connect does not return anything, you should not have `boardSerial` here

File: client/index.js [hunk 3]
```diff
@@ -6,7 +6,6 @@ module.exports = function() {
 
     // Factory Methods
     require('./components/board/boardFactory.js');
-    require('./components/board/simulatorFactory.js');
```
Comment (andrewjaykeller) [inline client/index.js:9]: victory!

File: client/components/board/boardFactory.js (hunk 2 above)
Comment (andrewheusser) [inline client/components/board/boardFactory.js]: ah good call!

File: client/components/board/boardFactory.js (hunk 1 above)
Comment (andrewheusser) [inline client/components/board/boardFactory.js]: nope, i dont!

**Comment by andrewheusser** [comment]
@pushtheworldllc made those changes, and also added in the experiment page I was working on from Friday

File: client/components/board/boardFactory.js [hunk 4]
```diff
@@ -39,16 +41,25 @@ angular.module('OpenEXP')
 
         // connect to the board and start streaming if no errors
         var connect = (portName) => {
+          return new Promise((res, rej) => {
             ourBoard.connect(portName)
-                .then(boardSerial => {
+                .then(() => {
```
Comment (teonbrooks) [inline client/components/board/boardFactory.js:46]: why is this empty?

File: client/components/board/boardFactory.js [hunk 5]
```diff
@@ -39,16 +41,25 @@ angular.module('OpenEXP')
...
                     ourBoard.on('ready', () => {
                         ourBoard.streamStart();
                         console.log('Connected!')
+                        counter = 0;
+                        ourBoard.on('sample', (sample) => {
+                          // wstream.write(sample.auxData[0] + ',' + sample.auxData[2] + '\n');
+                          // console.log(sample.channelData[0] + '\n')
+                          storage[counter] = sample;
+                          counter+=1;
```
Comment (teonbrooks) [inline client/components/board/boardFactory.js:55]: is this to know the total number of samples of the recording session? just curious, do we use this anywhere?

File: client/components/board/boardFactory.js [hunk 6]
```diff
@@ -39,16 +41,25 @@ angular.module('OpenEXP')
...
+                        })
                     })
                 }).catch((err) => {
                   console.log("error!")
                 })
-        };
+        })
```
Comment (teonbrooks) [inline client/components/board/boardFactory.js]: indent

**Comment by teonbrooks** [comment]
other than those, LGTM

File: client/components/board/boardFactory.js (hunk 4 above)
Comment (andrewheusser) [inline client/components/board/boardFactory.js:46]: because connect does not return anything

File: client/components/board/boardFactory.js (hunk 5 above)
Comment (andrewheusser) [inline client/components/board/boardFactory.js:55]: no its just to give each sample a unique value.  ultimately, what we'd like to do is turn the storage object into a buffer so we don't overload the app with data as its running.  the buffer could be written to a file as the data gets pushed out of the buffer

**Comment by teonbrooks** [comment]
thanks @andyh616!

**Pull request merged**